go.mod

```diff
@@ -252,6 +254,8 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.5.0
 )
 
+require github.com/docker/cli v27.5.0+incompatible
```

Comment:
This needs moving into the group above.

Reply:
Doh! Can't tell if it's my editor or `go mod tidy` doing this.